dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -231,7 +231,7 @@ def install(self):
             # https://github.com/tensorflow/tensorflow/issues/17316
             # Do not remove it.
             # the latest TF in conda needs to include /PREFIX/include
-            root_include = "-I/" + os.getenv("PREFIX") + "/include"
+            root_include = "-I/" + os.getenv("PREFIX", default="") + "/include"
```

Comment:
```suggestion
            root_include = "-I" + os.getenv("PREFIX", default="") + "/include"
```
Shouldn't the prefix contain the leading `/` if it specifies an absolute path?